NOM_visualizations/python/nom_data.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<p>Change "omics processing" to "data generation" And change variable (mentioned in comment above too) for <code>omicsprocess_has_input</code> to allude to data generation instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>